app/dashboard/templates/bulk_payout_bounty.html

```diff
@@ -215,7 +223,36 @@ <h5>{% trans 'Payout Preview' %}</h5>
       .entry.active{
         background-color: #bbb;
       }
-
+      .overageAlert {
```

Comment:
Hi @Anish-Agnihotri. Can you please move these styles and the other existing styles to a separate`.css` file? Else it's gonna add a lot of tech-debt later. Thanks in advance. :)

Reply:
For sure, @thelostone-mc suggested the same :).